src/mixins/document-mixin.ts

```diff
@@ -4,8 +4,8 @@ import { AxiosResponse } from 'axios'
 import { AxiosInstance as axios } from '@/utils'
 import { DocumentUpload, PdfInfoIF } from '@/interfaces'
 import { PdfPageSize } from '@/enums'
-import pdfjsLib from 'pdfjs-dist/build/pdf'
-pdfjsLib.GlobalWorkerOptions.workerSrc = require('pdfjs-dist/build/pdf.worker.entry')
+// import pdfjsLib from 'pdfjs-dist/build/pdf'
+// pdfjsLib.GlobalWorkerOptions.workerSrc = require('pdfjs-dist/build/pdf.worker.entry')
```

Comment:
This package has a webpack conflict so I deleted it temporarily.

Reply:
Just update it, it's what I did in `business-edit-ui`

`    "pdfjs-dist": "2.6.347",`

Reply:
Note: haven't tested

Reply:
^^ That doesn't work. Getting the following when trying to upload a PDF:

![image](https://user-images.githubusercontent.com/10002889/235724983-0d1228d9-95ba-4692-8fdd-54b950ed22c9.png)

However, this works:
```
// import library and create worker
// NB: do not convert require to an import
const pdfjsLib = require('pdfjs-dist/build/pdf.js')
pdfjsLib.GlobalWorkerOptions.workerPort = new Worker(
  new URL('pdfjs-dist/build/pdf.worker.js', import.meta.url)
)
```
then
```
    ...
    const data = new Uint8Array(arrayBuffer) // put it in a Uint8Array
    const document = await pdfjsLib.getDocument({ data }).promise // note use of "promise"
    for (let pageNum = 1; pageNum <= document.numPages; pageNum++) 
    ...
```

Reply:
However, the unit tests are unhappy with the above, working code:

[tests.log](https://github.com/bcgov/business-create-ui/files/11375465/tests.log)